end_to_end/gemma/7b/2_test_gemma.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a v4-16 and documentation for how to get started with Gemma-7b. 
+# Please make sure you have run end_to_end/gemma/7b/1_test_gemma.sh before running commands from this file. 
+
+# The flow of this file is as follows:
+# 1. Run decoding, finetuning of Gemma 7B with the converted checkpoint. Also, run pretraining of Gemma 7B
```

Comment:
nit: "1. Run decoding, finetuning of Gemma 7B with the converted checkpoint obtained from running end_to_end/gemma/7b/1_test_gemma.sh"